### DISCUSSION TITLE
multiple parallel  backups?

### DISCUSSION THREAD

**Discussion by sejtam** [outdoorbits/little-backup-box#212] [Q&A]
Can LBB manage multiple parallel backups? 
eg I insert 3 SD cards from 3 cameras (maybe 2 via an external reader, one into an USB port with a simple card reader) and then it copies them all in parallel?
Will I need to start them all at the same time, or can they be backgrounded and then the next dealt with?
or is there even a mode where it detects the sources and starts each backup automaticallly?

**Comment by outdoorbits** [comment]
This isn't possible in one step yet. Your will have to backup one by one.
It would be too complicated to manage multiple processes at the same time, even more when sharing one target device. I can't imagine this as a really save process. Over that, monitoring multiple processes wouldn't be possible on the small display.
In future I can imagine to add stack backups, one by one automatically. Actually I'm transferring the code to python. This will provide a more clear structure of the code. It's on my to-do. So please be patient a bit...

**Comment by sejtam** [comment]
Sure it may not be a overall time saver, but it would allow one to only have to set up once; and can then go do other things (eat, sleep) while for sequential backups one needs to pay attention to it again.

If you have a feature voting system where this is already listed, where can i +1 that?

**Comment by outdoorbits** [comment]
There is no feature voting system yet. ;-)
It's worth to try but raspberrys USB power is very limited and most probably such a solution would require a powered USB hub.
